server/src/models/userModel.ts

```diff
@@ -25,6 +56,12 @@ export const genToken = (id: string) => {
 };
 
 export const UserSchemaValidation = Joi.object({
-  name: Joi.string().min(2).max(25).required(),
-  email: Joi.string().min(2).max(30).required().email(),
+  firstName: Joi.string().min(2).max(25),
+  lastName: Joi.string().min(2).max(25),
+  email: Joi.string().min(5).max(30).email(),
+  password:Joi.string().optional(),
```

Comment:
we wont be using passwords. maybe just in the beginning. either way we will have to encrypt the password in a seperate collection if we do.

Reply:
yeah i know the password stuff is just there temporarily until we figure out the whole onBoarding signing in process